server/src/main/java/org/solarbank/server/service/CalculateService.java

```diff
@@ -0,0 +1,35 @@
+package org.solarbank.server.service;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.solarbank.server.dto.UserInputDto;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CalculateService {
+    /**
+     * temporary method to return static mock data.
+     *
+     * @param userInputDto data from user input
+     * @return response returns calculated data
+     */
+    public Map<String, Object> processUserInput(UserInputDto userInputDto) {
```

Comment:
Instead of passing in the full user request it's better to pass in the properties from the request that we require for completing the calculation.